src/foundry/prosemirror/schema/core.d.mts

```diff
@@ -0,0 +1,51 @@
+import { Node } from "prosemirror-model";
+
+export declare const paragraph: {
+  attrs: Record<string, any>;
+  managed: Record<any, any>;
+  content: string;
+  group: string;
+  parseDOM: Record<any, any>[];
```

Comment:
This should probably be `Record<string, unknown>[]`. A `Record<any, any>` allows a lot more than you would expect, for example arrays as well as functions because to JS/TS those are both ultimately both objects.